tests/conftest.py

```diff
@@ -33,6 +33,188 @@ def new_dir(tmpdir):
     os.chdir(cwd)
 
 
+@pytest.fixture()
```

Comment:
do we need this fixture? it is only used for the `test_cli.py::test_run_init_django`, but the init option does not look like it checks it. Maybe a spread test could be more appropriate that checks everything?